src/main/java/io/jenkins/plugins/orka/client/ConfigurationRequest.java

```diff
@@ -26,11 +26,24 @@
     @SuppressFBWarnings("URF_UNREAD_FIELD")
     private int cpuCount;
 
+    @SuppressFBWarnings("URF_UNREAD_FIELD")
+    private String scheduler;
+
     public ConfigurationRequest(String vmName, String image, String baseImage, String configTemplate, int cpuCount) {
         this.vmName = vmName;
```

Comment:
This ctor can/should call the new one with the `this` statement

Reply:
This way you have a single ctor doing all the initialization

Reply:
Unresolving this one as well. This ctor here needs to call the new one, not the other way around